test/vm_helpers.go

```diff
@@ -139,7 +139,7 @@ type testVM struct {
 
 	sshNetwork     string
 	macAddress     string // for SSH over TCP
-	port           int
+	port           uint
```

Comment:
This file changes doesn't  look like part of this commit.

Reply:
github UI does not make it easy to see which commit you were referring to :-/
There were some changes in the `gha:setup-go: Use go version from go.mod` which did not belong there, I moved these to the right commit (`lint: Fix potential vsock port overflows`).